pkg/cgroups/cgroups.go

```diff
@@ -0,0 +1,155 @@
+package cgroups
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// map of cgroup subsystems to their respective paths
+type Cgroups map[string]string
+
+// Find retrieves the mount points for all cgroup subsystems on the host. The
+// result of this operation should be cached if possible.
+func Find() (Cgroups, error) {
```

Comment:
stolen from https://github.com/docker/libcontainer/blob/master/cgroups/utils.go#L18 https://github.com/docker/libcontainer/blob/master/cgroups/utils.go#L101 https://github.com/measure/os/blob/master/misc/misc.go#L74

not yet tested, parsing all those files is probably expensive and the cgroup path is pretty predictable in our system so I didn't bother using this.